receiver/vcenterreceiver/client.go

```diff
@@ -464,17 +436,15 @@ func (vc *vcenterClient) vsanQueryByCluster(
 // VSANHosts returns back host VSAN performance metrics for a group of clusters
 func (vc *vcenterClient) VSANHosts(
 	ctx context.Context,
-	clusterRefs []*vt.ManagedObjectReference,
 ) (*VSANQueryResults, error) {
-	return vc.vSANQuery(ctx, VSANQueryTypeHosts, clusterRefs)
+	return vc.vSANQuery(ctx, VSANQueryTypeHosts)
```

Comment:
@BominRahmani unless you use the handleVSANError func inside here, you'll never turn any of the "lesser" errors to logs (same for the VM func)

Reply:
resolve